cocotb/decorators.py

```diff
@@ -238,19 +284,24 @@ def handle(self, record):
     def __init__(self, inst, parent):
         self.error_messages = []
         RunningCoroutine.__init__(self, inst, parent)
-        self.log = SimLog("cocotb.test.%s" % self.__qualname__, id(self))
+        self.log = SimLog("cocotb.test.%s" % inst.__qualname__, id(self))
         self.started = False
         self.start_time = 0
         self.start_sim_time = 0
         self.expect_fail = parent.expect_fail
         self.expect_error = parent.expect_error
         self.skip = parent.skip
         self.stage = parent.stage
-        self._id = parent._id
+        self._test_id = parent._id
```

Comment:
What's the reason for this change? I don't see it anywhere else.

Reply:
It's an artifact from when all tests were initialized during test discovery, leftover after rebase.

I wanted to sort based on the id's of the `test` object not the id of the created `RunningTask`, so each `RunningTest` would have an `_id` and a `_test_id`, and `RegressionManager` would sort via the latter. I'll remove it as it's no longer needed.